main.js

```diff
@@ -0,0 +1,17 @@
+const addButton = document.getElementById('add');
+const itemsContainer = document.getElementById('itemsContainer');
+const inputField = document.getElementById('inputField');
+
+addButton.addEventListener('click', () => {
+  const paragraph = document.createElement('p');
+  paragraph.classList.add('p-style');
+  paragraph.innerText = inputField.value;
+  itemsContainer.appendChild(paragraph);
+  inputField.value = '';
+  paragraph.addEventListener('click', () => {
```

Comment:
- [ ] Please let's add an event listener on a checkbox and not on the paragraph text. This means that you will require a checkbox button so that when the user clicks it, a checkmark in the box should indicate a completed task. Again when the user clicks on that same button, the checkmark should be removed indicating that the task is uncompleted.